src/Goerli-DssSpellCollateral.sol

```diff
@@ -29,12 +29,22 @@ contract DssSpellCollateralAction {
     //
     // A table of rates can be found at
     // https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
-    //
-    // uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978;
 
     // --- Math ---
-    // uint256 internal constant MILLION  = 10 ** 6;
-    // uint256 internal constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION                  = 10 ** 6;
+
+    uint256 constant FIFTY_PCT_RATE           = 1000000012857214317438491659;
+
+    // RWA007-A
+    bytes32 constant RWA007_A                 = "RWA007-A";
+    uint256 constant RWA007_A_AUTOLINE_AMOUNT = 500 * MILLION;
+    uint256 constant RWA007_A_AUTOLINE_GAP    = 100 * MILLION;
+    uint256 constant RWA007_A_AUTOLINE_TTL    = 1 weeks; 
```

Comment:
Why are those cached locally?
They also miss the variable visibility.
Consider passing them directly.

Reply:
cached?

Reply:
And btw they are internal by default, which is fine.

Reply:
I agree they could be added directly, I'd prefer that too, but the end result bytecode is the same. So I guess it's up to the spell crafter that decision.

Reply:
sorry, meant to say declared

Reply:
right, but we usually specify visibility as per PR checklist

Reply:
Anyway as I'm re-structuring this collateral file, I'm moving some things and probably trying to condense the code a bit, so might be doing this change.

Reply:
Just to visually verify the correctness of those params
https://github.com/makerdao/spells-goerli/blob/master/archive/2022-10-19-DssSpell/Goerli-DssSpellCollateral.sol